xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/store/XWikiHibernateStore.java

```diff
@@ -2064,8 +2064,10 @@ private void releaseAllLocksForCurrentUser(final XWikiContext ctx)
 
             // the select clause is compulsory to reach the fullName i.e. the page pointed
             Query query = session
-                .createQuery("select backlink.fullName from XWikiLink as backlink where backlink.id.link = :backlink");
+                .createQuery("select backlink.fullName from XWikiLink as backlink where backlink.id.link = :backlink"
```

Comment:
I'm wondering if using `backlink.id.link in (:backlink, :backlinkwithwiki)` wouldn't be better for performances than two different conditions.

Reply:
I can change if you think it can be better. I don't know much in term of perf there.
Now you think this change is safe and won't break anything?

Reply:
Looks good to me if you are sure the reference from others wiki end up in that table (I vaguely remember it was not the case at some point).